core/amber/src/main/resources/python_udf/demo_map_udf.py

```diff
@@ -0,0 +1,3 @@
+def map_function(row, args):
```

Comment:
please include some comment and explain the input and output of this function with an example.

Reply:
OK, will do.